### PR TITLE
Nit fixes and formatting following #62490 comments

### LINKED ISSUE #63792: Followup for #62490 to address all the comments
See https://github.com/elastic/elasticsearch/pull/62490#event-3884913243
for the last comments to address

### PR DESCRIPTION
**Pull request by BigPandaToo**
Resolves: #63792

### CODE REVIEW

**Review by ywangd — APPROVED** [review]
LGTM pending following comments. Also when you merge the PR, can we have a bit more descriptive title and commit message? Thanks!

File: x-pack/docs/en/rest-api/security/get-tokens.asciidoc [hunk 1]
```diff
@@ -125,21 +125,21 @@ seconds) that the token expires in, and the type:
...
+    "enabled" : true,
+    "authentication_realm" : {
+      "name" : "file",
+      "type" : "file"
+    },
+    "lookup_realm" : {
+      "name" : "file",
+      "type" : "file"
   },
   "authentication_type" : "realm"
```
Comment (ywangd) [inline x-pack/docs/en/rest-api/security/get-tokens.asciidoc:144]: These two lines are still not aligned.

File: x-pack/docs/en/rest-api/security/get-tokens.asciidoc [hunk 2]
```diff
@@ -125,21 +125,21 @@ seconds) that the token expires in, and the type:
...
+    "enabled" : true,
+    "authentication_realm" : {
+      "name" : "file",
+      "type" : "file"
+    },
+    "lookup_realm" : {
+    "name" : "file",
+    "type" : "file"
   },
   "authentication_type" : "realm"
```
[the remark this answers is not in the document]
Comment (BigPandaToo) [reply x-pack/docs/en/rest-api/security/get-tokens.asciidoc]: Done

File: x-pack/plugin/security/src/internalClusterTest/java/org/elasticsearch/xpack/security/authc/TokenAuthIntegTests.java [hunk 3]
```diff
@@ -134,7 +134,8 @@ public void testTokenServiceCanRotateKeys() throws Exception {
             assertNotNull(userTokenFuture.actionGet());
             assertNotEquals(activeKeyHash, tokenService.getActiveKeyHash());
         }
-        assertNotNull(response.getAuthenticationResponse());
+        assertNotNull(response.getAuthentication());
+        assertEquals(response.getAuthentication().getUser().getUsername(), SecuritySettingsSource.TEST_USER_NAME);
```
[the remark this answers is not in the document]
Comment (BigPandaToo) [reply x-pack/plugin/security/src/internalClusterTest/java/org/elasticsearch/xpack/security/authc/TokenAuthIntegTests.java]: Done

File: x-pack/plugin/security/src/internalClusterTest/java/org/elasticsearch/xpack/security/authc/pki/PkiAuthDelegationIntegTests.java [hunk 4]
```diff
@@ -149,7 +149,7 @@ public void testDelegateThenAuthenticate() throws Exception {
                         optionsBuilder.build());
                 String token = delegatePkiResponse.getAccessToken();
                 assertThat(token, is(notNullValue()));
-                assertNotNull(delegatePkiResponse.getAuthenticationResponse());
+                assertNotNull(delegatePkiResponse.getAuthentication());
```
[the remark this answers is not in the document]
Comment (BigPandaToo) [reply x-pack/plugin/security/src/internalClusterTest/java/org/elasticsearch/xpack/security/authc/pki/PkiAuthDelegationIntegTests.java:152]: Done

File: x-pack/plugin/security/src/internalClusterTest/java/org/elasticsearch/xpack/security/authc/TokenAuthIntegTests.java [hunk 5]
```diff
@@ -367,7 +368,7 @@ public void testRefreshingToken() throws IOException {
 
         // Assert that we can authenticate with the refreshed access token
         assertAuthenticateWithToken(refreshResponse.getAccessToken(), SecuritySettingsSource.TEST_USER_NAME);
-        assertNotNull(refreshResponse.getAuthenticationResponse());
+        assertNotNull(refreshResponse.getAuthentication());
```
[the remark this answers is not in the document]
Comment (BigPandaToo) [reply x-pack/plugin/security/src/internalClusterTest/java/org/elasticsearch/xpack/security/authc/TokenAuthIntegTests.java:371]: Done

File: x-pack/docs/en/rest-api/security/get-tokens.asciidoc (hunk 1 above)
Comment (BigPandaToo) [reply x-pack/docs/en/rest-api/security/get-tokens.asciidoc]: Fixed

**Pull request merged**